boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -50,8 +50,8 @@ public void testForNewLinkageErrors()
     Bom bom = RepositoryUtility.readBom(bomFile);
 
     ImmutableSetMultimap<SymbolProblem, ClassFile> oldProblems =
-        LinkageChecker.create(baseline).findSymbolProblems();
-    LinkageChecker checker = LinkageChecker.create(bom);
+        LinkageChecker.create(baseline, null).findSymbolProblems();
```

Comment:
Might be worth having an overloaded method to avoid the null argument.

Reply:
Created overloaded method.

What do you think about applying builder pattern? Created https://github.com/GoogleCloudPlatform/cloud-opensource-java/issues/1321